test/Controller/QueueControllerTest.php

```diff
@@ -0,0 +1,171 @@
+<?php
+/*
+ * This file is part of pmg/pheanstalk-bundle
+ *
+ * Copyright (c) PMG <https://www.pmg.com>
+ *
+ * For full copyright information see the LICENSE file distributed
+ * with this source code.
+ *
+ * @license     http://opensource.org/licenses/Apache-2.0 Apache-2.0
+ */
+
+namespace PMG\PheanstalkBundle;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\HttpFoundation\Request;
+use Pheanstalk\PheanstalkInterface;
+use PMG\PheanstalkBundle\Controller\QueueController;
+use PMG\PheanstalkBundle\Service\StatsService;
+
+class QueueControllerTest extends TestCase
+{
+    private $controller, $stats;
+
+    public function testControllerCanFetchAValidConnection()
+    {
+        $container = $this->loadContainer('default.yml');
+        $request = Request::createFromGlobals();
```

Comment:
Use `Request::create` here instead of `createFromGlobals`. `createFromGlobals` is more for the main entrypoint of the application.